core/src/test/java/kafka/test/ClusterTestExtensionsTest.java

```diff
@@ -84,37 +85,47 @@ public void testClusterTest(ClusterInstance clusterInstance) {
     public void testClusterTemplate() {
         Assertions.assertEquals(Type.ZK, clusterInstance.type(),
             "generate1 provided a Zk cluster, so we should see that here");
-        Assertions.assertEquals("Generated Test", clusterInstance.config().name().orElse(""),
```

Comment:
Please use `tags` to rewrite this assert

Reply:
thanks for the comment! let me add that